docs/index.md

```diff
@@ -2262,20 +2305,21 @@ memory and slows the search, especially for logging level 2.
 If you set logging level 2, you should restrict it such as by
 setting the label or setting the maximum number of branches.
 
-You can speed up searches by not allowing new disjoints, new statements,
+You can speed up searches by not allowing new disjoints, new statements/steps,
 and/or new variables, but in some cases this may mean a proof won't be
 found.
 
 This dialogue can be used to implement functions similar to
 certain functions of the mmj2 tool:
 
 * In mmj2 you can select a specific set of steps
-  that must be used in a justification of a given statement.
+  that must be used in a justification of a given statement/step.
   You can do something similar by doing a bottom-up
-  proof of that given statement, and selecting just those statements to be
-  used as root statements at the "first level".
-  This can quite similar to mmj2 if you also uncheck "allow new statements",
-  which will prevent the introduction of new statements.
+  proof of that given statement/step, and selecting just those statements/steps
+  to be used as root statements/steps at the "first level".
+  This can quite similar to mmj2 if you also uncheck
```

Comment:
probably a typo `This can quite similar to mmj2` -> `This can be quite similar to mmj2`